Week1/homework/js-exercises/6_array.js

```diff
@@ -0,0 +1,8 @@
+"use strict";
+let items = [];
+console.log("Empty Array:");
+console.log(items);
+let animals = ["lion", "tiger", "jaguar"];
+console.log(`${animals[1]} is the second element..`);
```

Comment:
the instructions asked to log the second array, not the second item in te array.